src/models/methods/ping.ts

```diff
@@ -1,11 +1,9 @@
-import { BaseRequest, BaseResponse } from './baseMethod'
+import type { BaseRequest, BaseResponse } from './baseMethod'
 
 export interface PingRequest extends BaseRequest {
   command: 'ping'
 }
 
 export interface PingResponse extends BaseResponse {
-  // TODO: figure out if there's a better way to type this
-  // eslint-disable-next-line @typescript-eslint/ban-types -- actually should be an empty object
-  result: {}
+  result: { role: string; unlimited: boolean }
```

Comment:
```suggestion
  result: { role?: string; unlimited?: boolean }
```
I think it only has those fields when it's rippled in standalone mode, because normal rippled doesn't have anything: https://xrpl.org/websocket-api-tool.html#ping